src/Pim/Bundle/CatalogBundle/tests/integration/Elasticsearch/PimCatalogTextAreaIndexConfigurationIntegration.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch;
+
+/**
+ * This integration tests checks that given an index configuration and some products indexed
+ * the text area research are consistent.
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class PimCatalogTextAreaIndexConfigurationIntegration extends AbstractIndexConfigurationIntegration
```

Comment:
right, we can put them  in `namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch\IndexConfiguration`

and call the test `PimCatalogTextAreaIntegration`